webhook/resourcesemantics/validation/validation.go

```diff
@@ -238,21 +245,27 @@ func (ac *reconciler) validate(ctx context.Context, req *admissionv1beta1.Admiss
 		return errMissingNewObject
 	}
 
-	if err := validate(ctx, newObj); err != nil {
+	if err := newObj.Validate(ctx); err != nil {
 		logger.Errorw("Failed the resource specific validation", zap.Error(err))
 		// Return the error message as-is to give the validation callback
 		// discretion over (our portion of) the message that the user sees.
 		return err
 	}
 
-	return nil
-}
+	// Generically callback if any are provided for the resource.
+	if callback, ok := ac.callbacks[gvk]; ok {
+		var unstruct unstructured.Unstructured
+		unstruct.NewEmptyInstance()
```

Comment:
This is just a copy ctor, methinks? I'd just remove this line

Reply:
Ooops sorry that was outdated. It's removed.